src/main/java/org/isf/generaldata/ExaminationParameters.java

```diff
@@ -21,21 +21,15 @@
  */
 package org.isf.generaldata;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.util.Properties;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 /**
  * Settings for Examination form
  *
  * @author Mwithi
  */
-public class ExaminationParameters {
+public class ExaminationParameters extends ConfigurationProperties {
 	
-	private final Logger logger = LoggerFactory.getLogger(this.getClass());
+	private static final String FILE_PROPERTIES = "examination.properties";
+	private final static boolean EXIT_ON_FAIL = false;
```

Comment:
The typical order of modifiers is `private static final <datatype> <variableName>`;  all the EXIT_ON_FAIL definitions across the files are the same (obviously cut-n-paste).

Why isn't the field defined once in the base class (`ConfigurationProperties`) and set to `false` as there is only one occurrence that is set to `true` (in `Version.java`)?

Reply:
> The typical order of modifiers is `private static final <datatype> <variableName>`; all the EXIT_ON_FAIL definitions across the files are the same (obviously cut-n-paste).

not exactly (it was copy-and-paste 😝) thanks!

> Why isn't the field defined once in the base class (ConfigurationProperties) and set to false as there is only one occurrence that is set to true (in Version.java)?

It was easier using the same pattern (still copy-and-paste) and was making clear the behavior for the single configuration, but thanks for the catch, I've modified now